README.md

```diff
@@ -77,12 +77,64 @@ We follow the following interface wherever possible to make it easy to swap data
 }
 ```
 
-### Why not Huggingface Datasets?
+## Adding a new dataset
+
+For general instructions, see the
+[TFDS guide to writing custom datasets](https://github.com/tensorflow/datasets/blob/master/docs/add_dataset.md).
+Instructions below are specific to this repository.
+
+Make a new folder inside `sign_language_datasets/datasets` with the same name as the dataset. As a convention, the name
+of the dataset should be lowercase and words should be separated by an underscore. Example:
+
+```sh
+mkdir sign_language_datasets/datasets/new_dataset
```

Comment:
This should be using the `tfds` cli: `tfds new my_dataset` - https://www.tensorflow.org/datasets/add_dataset